.github/workflows/go-build-and-publish.yml

```diff
@@ -0,0 +1,38 @@
+name: go-build-and-publish
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.2
```

Comment:
We may want to change this to what is currently the latest.  gov1.20.5

Reply:
Will do it